_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+ * Provide the range selection variation when a user needs to select both a start and an end date. 
+ * Pair the date picker pattern with data sets that extend across a time period to allow users to select the currently viewed data.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-hero-image.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+## Specifications
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Default layout
+The date picker component is composed of multiple buttons with four primary functions.
+
+ * Move the element to the next month
+ * Move the element to the previous month
+ * Selecting a specific date
+ * Move between the month selector and date selector states.
+
+When the Date picker is triggered, it should default to the current month and year
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-specifications.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Button states
+Text and icons should always be centered vertically and horizontally.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption:"Date picker button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-button-states-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption:"Month picker button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-button-states-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Date picker spacing
+The date picker element appears below corresponding text input that controls it. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-spacing.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+## Date selection interactions
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Single date selection
+ * Keyboard tabbing onto the input field and/or clicking on it automatically open the date picker
+ * The date may be filled in by hand, according to the format expressed in the placeholder
```

Comment:
Period needed at the end of the sentence.

Reply:
👍